doc/api/errors.md

```diff
@@ -563,6 +570,53 @@ found [here][online].
   encountered by [`http`][] or [`net`][] -- often a sign that a `socket.end()`
   was not properly called.
 
+<a id="nodejs-error-codes"></a>
+## Node.js Error Codes
+
+<a id="ERR_INVALID_ARG_TYPE"></a>
+### ERR_INVALID_ARG_TYPE
+
+The `'ERR_INVALID_ARG_TYPE'` error code is used generically to identify that
```

Comment:
Got it, so we should focus on migrating the errors to use the new system with a reasonable `code` right now, and deal with `message` later, because after the migration it would be easier to change them anyway?